.github/workflows/enos-run.yml

```diff
@@ -40,6 +40,8 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ${{ inputs.go-version }}
+      - name: Install tparse
+        run: go install github.com/mfridman/tparse@v0.10.3
```

Comment:
If we use `make tools` here instead we don't have to version this tool in two places.